packages/installer/upgrade.sh

```diff
@@ -27,6 +27,35 @@ find_partitions() {
         echo "OEM partition cannot be found"
         exit 1
     fi
+
+    CURRENT=$(df $0 | tail -1 | gawk '{print $1}')
```

Comment:
I would probably base it on the current mount points rather than having it based on `df` output command order, but those are details we can later polish.

Reply:
yup, for now I tried to make it simple, so I'm just checking where the current script is coming from